app/src/main/res/layout/admin_auth_activity.xml

```diff
@@ -111,6 +111,7 @@
           android:layout_marginBottom="44dp"
           android:background="@{viewModel.isSubmitButtonActive ? @drawable/state_button_primary_background :@drawable/start_button_transparent_background}"
           android:clickable="@{viewModel.isSubmitButtonActive}"
```

Comment:
Do we still need to bind clickable if we're setting enabled?

Reply:
Removed `clickable` as it is not needed any more.